esmvaltool/diag_scripts/ocean/diagnostic_maps.py

```diff
@@ -105,11 +150,247 @@ def make_map_plots(
         plt.close()
 
 
+def make_map_contour(
+        cfg,
+        metadata,
+        filename,
+):
+    """
+    Make a simple contour map plot for an individual model.
+
+    Parameters
+    ----------
+    cfg: dict
+        the opened global config dictionairy, passed by ESMValTool.
+    metadata: dict
+        the metadata dictionairy
+    filename: str
+        the preprocessed model file.
+
+    """
+    # Load cube and set up units
+    cube = iris.load_cube(filename)
+    cube = diagtools.bgc_units(cube, metadata['short_name'])
+
+    # Is this data is a multi-model dataset?
+    multi_model = metadata['dataset'].find('MultiModel') > -1
+
+    # Make a dict of cubes for each layer.
+    cubes = diagtools.make_cube_layer_dict(cube)
+
+    # Load image format extention and threshold.thresholds.
+    image_extention = diagtools.get_image_format(cfg)
+
+    # Load threshold/thresholds.
+    plot_details = {}
+    colours = []
+    thresholds = diagtools.load_thresholds(cfg, metadata)
+
+    for itr, thres in enumerate(thresholds):
+        if len(thresholds) > 1:
```

Comment:
if thresholds: (no more checks on list length pls)

Reply:
This method is when there are more than one threshold. ie the list needs to be length **2** or more. 

```
        if thresholds:
```
is not the same as 
```
        if len(thresholds) > 1:
```

Reply:
ow yes, silly me